src/constants/tokens.ts

```diff
@@ -20,4 +20,12 @@ const zIndex = {
   dropdown: 50,
 } as const;
 
-export { size, zIndex };
+const fontSize = {
+  xs: "8px",
+  s: "12px",
+  m: "14px",
+  l: "16px",
+  xl: "18px",
+} as const;
```

Comment:
Interesting, this almost feels like _too_ many font sizes to me. Is this perhaps an opportunity to standardize some uses? E.g. move all 16px usages to either 14px or 18px